app/coffeescript/components/ui/base_map.coffee

```diff
@@ -91,16 +78,19 @@ define [
 
     @fireOurMapEvents = () ->
       eventsHash = @attr.gMapEvents
-      if @attr.infoWindowOpen == true
+      if @attr.infoWindowOpen is true
         eventsHash['center_changed'] = false
         eventsHash['max_bounds_changed'] = false
-      clearInterval(@intervalId)
 
       if eventsHash['max_bounds_changed']
         @trigger document, 'uiMapZoomForListings', @mapChangedData()
         @trigger document, 'uiInitMarkerCluster', @mapChangedData()
         @trigger document, 'mapRendered', @mapChangedData()
         @trigger document, 'uiNeighborhoodDataRequest', @mapChangedDataBase()
+      else if eventsHash['zoom_changed']
+        @trigger document, 'uiMapZoom', @mapChangedData()
+      else if eventsHash['center_changed']
+        @trigger document, 'uiMapCenter', @mapChangedData()
```

Comment:
:+1: I changed the `else if`'s to `if` but kept them separate events: `uiMapZoom` and `uiMapCenter`. I felt that adding another generic event would create more confusion with what is there now.